src/coprocessor/endpoint.rs

```diff
@@ -654,6 +731,31 @@ impl<E: Engine> Endpoint<E> {
     }
 }
 
+fn make_error_batch_response(batch_resp: &mut coppb::StoreBatchTaskResponse, e: Error) {
+    match e {
```

Comment:
Do we need a warn log and update the `COPR_REQ_ERROR` metrics like `make_error_response` here?

Reply:
I think so, related information is added.